src/components/layout/layout.js

```diff
@@ -159,14 +161,21 @@ const Layout = ({
         const website_status = useWebsiteStatusApi()
 
         React.useEffect(() => {
-            if (website_status) {
+            if (website_status && !isTestlink()) {
                 const current_client_country = website_status?.clients_country || ''
                 const client_information_cookie = new CookieStorage('client_information')
                 const residence = client_information_cookie.get('residence')
 
                 setRedirectionApplied(true)
                 handleRedirect(residence, current_client_country, window.location.hostname)
             }
+            if (isTestlink()) {
+                const testlink_url = isBrowser() && localStorage.getItem('config.testlink_url')
+                if (testlink_url) {
+                    setRedirectionApplied(true)
+                    handleRedirect(testlink_url)
+                }
+            }
```

Comment:
Keep the `layout` file clean. No need to add anything here. 

`handleRedirect` function will block the test link redirection accordingly